config/value.go

```diff
@@ -363,8 +396,7 @@ func convertValue(value interface{}, targetType reflect.Type) (interface{}, erro
 			return reflect.ValueOf(target).Elem().Interface(), err
 		}
 	}
-
-	return nil, fmt.Errorf("can't convert %v to %v", valueType, targetType)
+	return nil, nil
```

Comment:
1. It doesn't make sense to me to communicate through returned pointers(yeah, I know, I am weird :)
2. Could you please update the comments to the function then?